src/components/sign-in/sign-in.tsx

```diff
@@ -194,12 +194,22 @@ const SignIn = ({ dark, position }: Props) => {
           <img className={styles.unsupportedNetworkIcon} src={images.unsupportedNetwork} alt="network not supported" />
           <h5>Unsupported chain!</h5>
 
-          <p className={globalStyles.mtXl}>Supported networks are: {supportedNetworks}</p>
+          <p className={globalStyles.mtXl}>
+            Supported networks are:{' '}
```

Comment:
I would rather word this as `The following networks are supported:` or just `Supported networks:`